app/assets/javascripts/expand.js

```diff
@@ -20,13 +20,19 @@ $(document).ready(function() {
       var title = response.event.title;
       var description = response.event.description;
       var title_header = document.createElement("h1");
-      var para = document.createElement("p");
+      var para1 = document.createElement("p");
+      var para2 = document.createElement("p");
+      var para3 = document.createElement("p");
+      var para4 = document.createElement("p");
       var title_node = document.createTextNode(title);
       var desc_node = document.createTextNode(description);
+      var date_and_time = response.event.date_and_time
 
-
+      var break_line = document.createElement("br");
```

Comment:
Identifier 'break_line' is not in camel case.<br>Mixed double and single quotes.